.github/workflows/release.yaml

```diff
@@ -14,6 +14,8 @@ jobs:
     permissions:
       contents: write  # for Git to git push
       pull-requests: write # to allow creating PR
+      id-token: write # for npm provenance
```

Comment:
This new permission is necessary, according to the npm documentation: https://docs.npmjs.com/generating-provenance-statements#publishing-packages-with-provenance-via-github-actions